pkg/volume/csi/csi_plugin.go

```diff
@@ -182,6 +184,7 @@ func (h *RegistrationHandler) DeRegisterPlugin(pluginName string) {
 
 func (p *csiPlugin) Init(host volume.VolumeHost) error {
 	p.host = host
+	p.serviceAccountTokenGetter = host.GetServiceAccountTokenFunc()
```

Comment:
I guess this was here before, but since this function is only implemented for kubelet, it could be in the KubeletVolumeHost instead of the general VolumeHost.

Reply:
sorry, what is issue here? it is available in the general VolumeHost

Reply:
I meant since this is functionality is specific to kubelet (ie the other controllers don't implement this interface), then we could put it directly in KubeletVolumeHost instead of the generic VolumeHost.

Reply:
done